src/gui/qgsfilewidget.cpp

```diff
@@ -84,12 +84,14 @@ QStringList QgsFileWidget::splitFilePaths( const QString &path )
 #if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
   const QStringList pathParts = path.split( QRegExp( "\"\\s+\"" ), QString::SkipEmptyParts );
 #else
-  const QStringList pathParts = path.split( QRegExp( "\"\\s+\"" ), Qt::SkipEmptyParts );
+  const QStringList pathParts = path.split( QRegularExpression( "\"\\s+\"" ), Qt::SkipEmptyParts );
```

Comment:
Is there a reason why you didn't apply the const thread_local optimization to this static regular expression?